### PR TITLE
ENH: Add Pixi GitHub Actions Configuration

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

**Comment by thewtex** [comment]
@blowekamp @jhlegarreta additional pixi steps forward.

This adds C++ configure, build, test CI testing to exercise the pixi
configuration, exercise the associated conda-forge toolchains, and
improve coverage. And an ergonomic improvement.

Next steps include macos additions, investigate Python CI additions.

**Review by jhlegarreta — APPROVED** [review]
:1st_place_medal: Thanks for doing this @thewtex.

File: .github/workflows/pixi.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            - '*.md'
+            - LICENSE
+            - NOTICE
+            - 'Documentation/**'
+            - 'Utilities/Debugger/**'
+            - 'Utilities/ITKv5Preparation/**'
+            - 'Utilities/Maintenance/**'
+
+env:
+    ExternalDataVersion: 5.3.0
```
Comment (dzenanz) [inline .github/workflows/pixi.yml:27]: Should this be 5.4.0?

**Comment by thewtex** [comment]
\azp run ITK.Linux

**Pull request merged**